### PR TITLE
Add homepage tag back

### LINKED ISSUE #28: add homepage meta tag back
removed in #26 because we need to rework what we do when no homepage is provided

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds the homepage back

Requires https://github.com/axodotdev/axohtml/pull/8 to be merged

closes https://github.com/axodotdev/oranda/issues/28

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -5,11 +5,12 @@ version = "0.0.0"
 edition = "2021"
 license = "MPL-2.0"
 
+
```
Comment (ashleygwilliams) [inline Cargo.toml]: i think this is extraneous

File: src/site/html.rs [hunk 2]
```diff
@@ -6,14 +6,21 @@ pub fn build(config: &Config, content: String) -> String {
     let theme = theme::css_class(&config.theme);
     let classlist: &str = &format!("body container {}", theme)[..];
     let description = &config.description;
+    let homepage = config.homepage.as_ref().map(|homepage| {
```
Comment (ashleygwilliams) [inline src/site/html.rs:10]: does this resolve to an empty string if home page is `None`?

**Review by ashleygwilliams — COMMENTED** [review]
i think this looks good- just remove the extra space. am curious the answer to my question! def wish we had that inline conditional

File: src/site/html.rs (hunk 2 above)
Comment (SaraVieira) [reply src/site/html.rs:10]: from what I could tell yes, no homepage and no html is rendered

File: Cargo.toml (hunk 1 above)
Comment (SaraVieira) [reply Cargo.toml]: removed

**Review by ashleygwilliams — APPROVED** [review]
:shipit:

**Pull request merged**